### PR TITLE
Implicit relations fixes

### PR DESCRIPTION
**Pull request by mhwelander**

### CODE REVIEW

**Comment by mhwelander** [comment]
Closes #521, closes #414

File: content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx [hunk 1]
```diff
@@ -338,9 +328,9 @@ model Profile {
 
 ## The @relation attribute
 
-The `@relation` attribute can only be applied to the [relation fields](#relation-fields), not to [scalar fields](data-model#scalar-types).
+The `@relation` attribute can only be applied to the [relation fields](#relation-fields), not to [scalar fields](data-model#scalar-types). The attributes is required on **one**s side of 1-1 and 1-n relations.
```
Comment (janpio) [inline content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx]:
```suggestion
The `@relation` attribute can only be applied to the [relation fields](#relation-fields), not to [scalar fields](data-model#scalar-types). The attributes is required on **one** side of 1-1 and 1-n relations.
```

**Review by janpio — APPROVED** [review]
ahem, undid this approve after the fact. misclick.

**Review by janpio — COMMENTED** [review]
Looks good, but I do not know nearly enough about implicit relations to leave a ✔️

File: content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx [hunk 2]
```diff
@@ -164,35 +164,25 @@ Implicit many-to-many relations define relation fields as lists on both sides of
...
 
 model Category {
   id    Int     @id @default(autoincrement())
-  posts Post[]  @relation(references: [id])
+  posts Post[]
 }
 ```
 
-Note that both relation fields in a many-to-many relation must be annotated with the `@relation` attribute and _reference_ the fields of the other model they're mapping to.
+> **Note**: Implicit many-to-many relations do **not** require the `@relation` attribute.
```
Comment (mavilein) [inline content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx]: Mostly yes. Except when you are dealing with ambiguous relations. Then you need to provide the attribute with the name argument.

**Review by mavilein — APPROVED** [review]

File: content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx (hunk 2 above)
Comment (mhwelander) [reply content/03-reference/01-tools-and-interfaces/01-prisma-schema/06-relations.mdx]: Added link to 'disambiguate relations' and noted that it's not specific to implicit m-n.

**Comment by mhwelander** [comment]
Thanks both! Squash-merging in 3, 2, 2.5, 1... 🚀

**Pull request merged**